### PR TITLE
Twitch chat integration with inputs included

### PR DESCRIPTION
**Pull request by KoolieAid**
has effects:
 - Chat can choose to attack
 - Chat can temp disable inputs for 5 secs (lol)
 - Spawn 5 eggs on player

No UI cuz idk how

### CODE REVIEW

File: Assets/Scripts/Player Folder/Player.cs [hunk 1]
```diff
@@ -142,33 +150,12 @@ private void Attack()
...
-            default:
-                Debug.LogError("HOW");
-                break;
-        }
+        
+        if (counter == 1) attackParticle.Play();
+        var variation = counter == 1 ? "" : counter.ToString();
+        animator.ResetTrigger($"Attack Finish{variation}");
+        animator.SetTrigger($"Attack Start{variation}");
+        PlayRandomAttackSound();
```
Comment (KoolieAid) [inline Assets/Scripts/Player Folder/Player.cs:158]: @LauSnchzz lol look at the old version (that was probs urs idk forgot)

Comment (LauSnchzz) [reply Assets/Scripts/Player Folder/Player.cs:158]: yeah that aint me

Comment (LauSnchzz) [reply Assets/Scripts/Player Folder/Player.cs:158]: nvm it was me lmao

**Review by LauSnchzz — APPROVED** [review]
looks cool, unfortunately cant check in unity as i dont have it on my laptop

**Pull request merged**